templates/deploy-master-statefulset.yaml

```diff
@@ -135,6 +136,7 @@ spec:
               value: {{ include "postgresql.subchart" . }}
             - name: POSTGRESQL_PORT
               value: {{ (include "deploy.postgresql.service.port" .) | quote }}
+          resources: {{- include "common.resources.preset" (dict "type" "nano") | nindent 12 }}
```

Comment:
here and on other places for init-containers we should use constants from the values yaml for the waiting containers